src/pkg/bundle/deploy.go

```
@@ -153,7 +157,7 @@ func deployPackages(packages []types.Package, resume bool, b *Bundle) error {
 }
 
 // loadVariables loads and sets precedence for config-level and imported variables
-func (b *Bundle) loadVariables(pkg types.Package, bundleExportedVars map[string]map[string]string) map[string]string {
+func (b *Bundle) loadVariables(pkg types.Package, bundleExportedVars map[string]map[string]string) (map[string]string, error) {
```

do we need to return an error? looks like we only ever return nil for the error